.github/workflows/ci.yml

```diff
@@ -0,0 +1,100 @@
+on: push
+
+env:
+  DUCKDB_GIT_TAG: '662041e2b43283bfcfeed984af7c3eaa85fb4996'
```

Comment:
Instead of hard-coding commit hash, can you use tag?

Reply:
Done.